app/src/components/main/grid/gridHeader/gridHeader.jsx

```diff
@@ -41,6 +41,9 @@ export const GridHeader = ({
         key={column.id || i}
         title={column.title}
         align={column.align}
+        width={column.width}
```

Comment:
`width` sounds as absolute width value. I think that name f.e. `size` will be better.